src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -395,26 +406,46 @@ export const StartDate = ({
           />
         ) : (
           <>
-            {quotes.map((quote) => (
-              <DateForm
-                key={quote.id}
-                startDate={quote.startDate}
-                currentInsurer={quote.currentInsurer ?? undefined}
-                quoteDisplayName={quote.displayName}
-                dataCollectionId={quote.dataCollectionId ?? undefined}
-                modal={modal}
-                disabled={
-                  loadingQuoteIds.length > 0 ||
-                  Boolean(quote.currentInsurer && !quote.startDate)
-                }
-                isSplit={quoteBundleSelector.isMultiQuote(selectedBundle)}
-                size={size}
-                onChange={(newDate) =>
-                  handleSelectNewStartDate(newDate, [quote.id])
-                }
-                loading={loadingQuoteIds.includes(quote.id)}
-              />
-            ))}
+            {quotes.map((quote, index, arr) => {
+              const isArrayLengthEven = arr.length % 2 === 0
+              const isItemIndexEven = index % 2 === 0
+              const isMultiQuote = quoteBundleSelector.isMultiQuote(
+                selectedBundle,
+              )
```

Comment:
We don't need to check this anymore, right?

`isExpandedFirstItem === true` works for 1, 3, 5 items :)